components/tidb_query_datatype/src/error.rs

```diff
@@ -1,4 +1,7 @@
-// Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
I don't think many errors in this file should belong to `datatype`. I prefer put it in `common`.